### PR TITLE
Copyedits, round 1!

### PR DESCRIPTION
**Pull request by sarahmonster**
Needs more still, but this makes a few wee things better.

### CODE REVIEW

File: src/components/SiteFooter/index.js [hunk 1]
```diff
@@ -30,7 +30,15 @@ const SiteFooter = () => {
...
         {/* eslint-disable-next-line react/jsx-curly-brace-presence */}
         {` · `}
-        <Link to="/privacy">Privacy policy</Link>
+        <Link to="/privacy">Privacy</Link>
+        {/* eslint-disable-next-line react/jsx-curly-brace-presence */}
+        {` · `}
+        <Link to="/credits">Credits</Link>
+        {/* eslint-disable-next-line react/jsx-curly-brace-presence */}
+        {` · `}
+        <Link as="a" href="https://github.com/octopusthink/getmicdrop.com">
```
Comment (tofumatt) [inline src/components/SiteFooter/index.js]: I think we should remove the GitHub link, as it implies that the source for the actual app is available on GitHub, which isn't the case and makes this seem a bit misleading 😄

File: src/content/pages/credits.mdx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+title: Credits
+---
+
+import Panel from 'components/Panel';
+
+<Panel>
+
+
+# Photo credits
```
Comment (tofumatt) [inline src/content/pages/credits.mdx]: Won't the title be used as the H1 in the template? If not it probably should be! I'd think this should be an H2:
```suggestion
## Photo credits
```

File: src/content/pages/index.mdx [hunk 3]
```diff
@@ -1,16 +1,17 @@
 ---
-title: Want to take it for a spin?
+title: The menubar app for controlling your microphone.
```
Comment (tofumatt) [inline src/content/pages/index.mdx]: 👍

File: src/content/pages/index.mdx [hunk 4]
```diff
@@ -45,6 +46,18 @@ We're looking for beta testers to make sure everything works well before launch.
...
 
+<Spacer />
+
+<TextBlock end>
+
+## Now, get ready for your meetings to suck less.
+
+We can't promise to keep your overly-enthusiastic co-worker from cutting you off, but we _can_ make it a little bit easier to control your sound.
+
+All beta testers who send us feedback will receive a lifetime free-forever copy of the app, as a token of our appreciation.
```
Comment (tofumatt) [inline src/content/pages/index.mdx]:
```suggestion
All beta testers will receive a lifetime copy of the app, as a token of our appreciation.
```

File: src/components/SiteFooter/index.js (hunk 1 above)
Comment (sarahmonster) [reply src/components/SiteFooter/index.js]: I already fixed this last night! Just spaced and forgot to push.

File: src/content/pages/credits.mdx (hunk 2 above)
Comment (sarahmonster) [reply src/content/pages/credits.mdx]: Let's do this for now and re-evaluate the title/header setup in a separate PR—we may want some more differentiated styling for page titles too.

**Pull request merged**